<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunOptionsDialog</class>
 <widget class="QDialog" name="RunOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>588</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Define Run Options</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <item>
    <widget class="QGroupBox" name="storeTypeGroup">
     <property name="title">
      <string>Type Of Store</string>
     </property>
     <layout class="QGridLayout" name="typeStoreLayout">
      <property name="margin">
       <number>10</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="4" column="0" colspan="3">
       <widget class="QWidget" name="SQLWidget" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QLineEdit" name="SQLNameEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="SQLselectFile">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../tonatiuh.qrc">
             <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="SQLNameLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>    SQL BD file       </string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="datFileRadio">
        <property name="text">
         <string>.dat file</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="dataBaseRadio">
        <property name="text">
         <string>SQL</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QWidget" name="datWidget" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="fileStoreLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>10</number>
         </property>
         <item row="13" column="2">
          <widget class="QSpinBox" name="raysPerFileSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
           <property name="singleStep">
            <number>100000</number>
           </property>
           <property name="value">
            <number>100000</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="3">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Selecte the file name to store</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="selectSurfaceButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Choose surface...</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="12" column="2">
          <widget class="QSlider" name="raysPerFileSlider">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
           <property name="singleStep">
            <number>100000</number>
           </property>
           <property name="sliderPosition">
            <number>100000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QRadioButton" name="exportSurfaceRadioButton">
           <property name="text">
            <string>Export surface</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="6" column="5">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="exportAllPhotonsRadio">
           <property name="text">
            <string>Export All Photon Map:</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="7" column="0" colspan="5">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Select the number of files to store</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QRadioButton" name="oneFileRadio">
           <property name="text">
            <string>One file</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QLabel" name="raysperFileLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Max. number of photons per file to store:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QRadioButton" name="moreFilesButton">
           <property name="text">
            <string>More than one file</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="5">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Select photons to store:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLineEdit" name="surfacePathLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>File name:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLineEdit" name="fileNameEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="6" column="4">
          <widget class="QPushButton" name="selectFileSave">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../tonatiuh.qrc">
             <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="storeTypeLabel">
        <property name="text">
         <string>Store type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="storeTypeCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="storeInfoGroup">
     <property name="title">
      <string>Please, deselect the information you don't want to export:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QCheckBox" name="sideBox">
        <property name="text">
         <string>Surface Side</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="coordBox">
        <property name="text">
         <string>Coordinates</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="powerPerPhotonCheck">
        <property name="text">
         <string>Power Per Photon</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../tonatiuh.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RunOptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RunOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportSurfaceRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectSurfaceButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>raysPerFileSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>raysPerFileSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>raysPerFileSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>raysPerFileSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportSurfaceRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>surfacePathLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>datFileRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>datWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dataBaseRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>SQLWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moreFilesButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>raysperFileLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moreFilesButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>raysPerFileSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moreFilesButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>raysPerFileSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
